manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -295,7 +297,10 @@ public Long getProcessorsCount(String bridgeId, String customerId) {
     @Transactional
     @Override
     public ListResult<Processor> getUserVisibleProcessors(String bridgeId, String customerId, QueryProcessorResourceInfo queryInfo) {
-        Bridge bridge = bridgesService.getReadyBridge(bridgeId, customerId);
+        Bridge bridge = bridgesService.getBridge(bridgeId, customerId);
+        if (!isManagedResourceActionable(bridge)) {
+            throw new BridgeLifecycleException(String.format("Bridge with id '%s' for customer '%s' is not in READY/FAILED state.", bridge.getId(), bridge.getCustomerId()));
```

Comment:
this log may become a misleading if we change `isManagedResourceActionable` implementation, WDYT?

Reply:
keep it as it is